deep-learning/_posts/2017-04-04-feature-pyramid-network.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: review
+title:  "Feature Pyramid Networks for Object Detection"
+tags:   deep-learning CNN localization bounding-boxes
+author: Frédéric Branchaud-Charron
+pdf:   "https://arxiv.org/pdf/1612.03144.pdf"
+cite:
+  authors: "T. Lin, P. Dollár, R. Girshick, K. He, B. Hariharan, S. Belongie"
+  title:   "Feature Pyramid Networks for Object Detection"
+  venue:   "Computer Vision and Pattern Recognition, 2017. CVPR 2017"
+---
+
+The Feature Pyramid Network (FPN) looks a lot like the [U-net]({{ site.baseurl }}{% link deep-learning/_posts/2017-02-27-unet.md %}). The main difference is that there is multiple prediction layers: one for each upsampling layer.
+<div align="middle">
+  <img src="/deep-learning/images/fpn/architecture.png" width="400">
+</div>
+
+
+## FPN for region-proposal
+To achieve region-proposal, the authors add a 3x3 Conv layer followed by two 1x1 Conv for classification and regression on each upsampling layer. These additions are called **heads** and the weights are shared. For each head, you assign the sames anchors boxes, resized to match the head's shape. The ground truth labels are assigned to the anchor if it has >70% IoU.
```

Comment:
> For each head, you assign the sames anchors boxes, resized to match the head's shape. The ground truth labels are assigned to the anchor if it has >70% IoU.

Could you clarify this part? Same anchor boxes as what? What does assigning a _ground truth_ label to an anchor box mean?

Also, "sames" should be "same".